src/shared/js/finance/xmlDocumentToDocumentBudgetaire.js

```diff
@@ -30,10 +29,11 @@ export default function(doc, natureToChapitreFI){
             ret[key] = l.getElementsByTagName(key)[0].getAttribute('V')
         })
 
+        ret['OpBudg'] = Number(l.getElementsByTagName('OpBudg')[0].getAttribute('V'));
```

Comment:
If you haven't filtered OpBudg here you'll probably need to do it client-side

Reply:
Ce matin c'était pas sûr ; là j'ai eu confirmation que ça n'était pas intéressant.